### PR TITLE
B 21773 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
- must have "counselor_move_create" FF on
1. Access the Office app
2. Login as a services counselor
3. go to customer search- type some letters to get the "add customer" button to show up
4. Clcik Add Customer
5. validate the "Is this a Bluebark Move?" shows up
6. When yes is selected - the okta account and Non-Cac users forms are hidden
7.  When "safety_move_flag" is on and the user has safety Privs- validate that you can not have safety move and bluebark able to both be "Yes"
8. fill out the customer info submit
9. Validate on "Tell us about the orders" page that the bluebark orders type is prefilled and unchangeable
10. go back and select a diff order type- make sure that blueback can not be selected on this page 
11. 

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

## Screenshots
image
image

image
image
image

### CODE REVIEW

File: pkg/models/order_test.go [hunk 1]
```diff
@@ -485,3 +485,31 @@ func (suite *ModelSuite) TestSaveOrderWithoutPPM() {
...
+					OrdersType: internalmessages.OrdersTypePERMANENTCHANGEOFSTATION,
+				},
+			},
+		}, nil)
+
+		canSendEmail := order.CanSendEmailWithOrdersType()
+		suite.True(canSendEmail)
+	})
+
+	suite.Run("Safety and BB orders cannot send email", func() {
```
Comment (brianmanley-caci) [inline pkg/models/order_test.go:503]: This says Safety and BB, but it only tests the Blue Bark case. You could add a safety order and verify that doesn't send as well.

File: src/components/Office/AddOrdersForm/AddOrdersForm.jsx [hunk 2]
```diff
@@ -122,7 +129,7 @@ const AddOrdersForm = ({ onSubmit, ordersTypeOptions, initialValues, onBack, isS
                 name="ordersType"
                 options={ordersTypeOptions}
                 required
-                isDisabled={isSafetyMoveSelected}
+                isDisabled={isSafetyMoveSelected || isBluebarkMoveSelected}
```
Comment (brianmanley-caci) [inline src/components/Office/AddOrdersForm/AddOrdersForm.jsx:132]: You could add a test to the test file to verify this is disabled for safety or bluebark.

**Review by danieljordan-caci — COMMENTED** [review]
Screenshot 2024-12-17 at 8 30 09 AM

Any thoughts on having this outside of the affiliation box?

Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?

**Comment by deandreJones** [comment]
> Screenshot 2024-12-17 at 8 30 09 AM
> 
> Any thoughts on having this outside of the affiliation box?
> 
> Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?

rolling my eyes.... I don't hate that idea

**Comment by deandreJones** [comment]
> > Screenshot 2024-12-17 at 8 30 09 AM
> > Any thoughts on having this outside of the affiliation box?
> > Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?
> 
> rolling my eyes.... I don't hate that idea

how's this Danny boy

image

**Comment by danieljordan-caci** [comment]
> > > Screenshot 2024-12-17 at 8 30 09 AM
> > > Any thoughts on having this outside of the affiliation box?
> > > Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?
> > 
> > 
> > rolling my eyes.... I don't hate that idea
> 
> how's this Danny boy
> 
> image

It's beautiful!

**Comment by danieljordan-caci** [comment]
> > > > Screenshot 2024-12-17 at 8 30 09 AM
> > > > Any thoughts on having this outside of the affiliation box?
> > > > Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?
> > > 
> > > 
> > > rolling my eyes.... I don't hate that idea
> > 
> > 
> > how's this Danny boy
> > image
> 
> It's beautiful!

Except for some consistency with `Safety move` and `Bluebark Move` - just need to decide on `move` or `Move`

**Comment by deandreJones** [comment]
> > > > > Screenshot 2024-12-17 at 8 30 09 AM
> > > > > Any thoughts on having this outside of the affiliation box?
> > > > > Possibly have a "Special Moves" section above the affiliation where we can populate all of the bluebark/safety move things?
> > > > 
> > > > 
> > > > rolling my eyes.... I don't hate that idea
> > > 
> > > 
> > > how's this Danny boy
> > > image
> > 
> > 
> > It's beautiful!
> 
> Except for some consistency with `Safety move` and `Bluebark Move` - just need to decide on `move` or `Move`

did you put on your thick glasses to notice that

**Comment by danieljordan-caci** [comment]
> > Except for some consistency with `Safety move` and `Bluebark Move` - just need to decide on `move` or `Move`
> 
> did you put on your thick glasses to notice that

Gotta put my extra thick ones on when reviewing your spellings

**Comment by deandreJones** [comment]
> > > Except for some consistency with `Safety move` and `Bluebark Move` - just need to decide on `move` or `Move`
> > 
> > 
> > did you put on your thick glasses to notice that
> 
> Gotta put my extra thick ones on when reviewing your spellings

oomph low blow

**Review by traskowskycaci — APPROVED** [review]
Disclaimer - cannot run client so did not see UI changes myself BUT
Reviewed your code and tests against the BL's AC, as well as checked out the screenshots posted by you and Daniel.

**Review by brianmanley-caci — APPROVED** [review]
Everything is working well for me, and all previous comments were addressed.

**Pull request merged**